react/src/components/blocks/SearchForm/SearchForm.js

```diff
@@ -1,5 +1,56 @@
+import React, { useRef, useState, useEffect } from 'react'
+
+// ui
+import SettingsIcon from '@/ui/icons/SettingsIcon'
+import AutoCompleteInput from '@/ui/Fields/AutoCompleteInput'
+import Controls from './Controls'
+
+// image import
+import GoogleLogo from '../../../static/images/google_logo.png'
+
+// styles
+import styles from './SearchForm.module.css'
+
 function SearchForm() {
-  return <p>Search Form component</p>
+  const [down, setDown] = useState(false)
+  const prevScrollY = useRef(0)
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const currentScrollY = window.scrollY
+
+      prevScrollY.current = currentScrollY
+
+      setDown(currentScrollY > 300)
+    }
+
+    window.addEventListener('scroll', handleScroll, { passive: true })
+
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [])
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.mainContainer}>
+        <div className={styles.formContainer}>
+          <div className={styles.logo}>
+            <a href="#" title="Go to Google Home" id="logo" data-hveid="8">
```

Comment:
localize

Reply:
@ramsunvtech done.

Reply:
@naveenvignesh5 hope you are working with updated code. This is already done